### ISSUE TITLE
Html Audio Control: Can't set currentTime

### ISSUE THREAD

**Issue by Terrence77** [MicrosoftEdge/WebView2Feedback#3422]
**Description**
When adding an html <audio> control to a razor page in a WinForms app,
you are unable to advance the time bar, which corresponds to the audio.currentTime property.

The same bug exists in a Maui Blazor App
The bug does not exist in a regular Blazor Server App.

![image](https://user-images.githubusercontent.com/46733900/233742716-c30ffa66-b4c0-4f85-8e16-33b6ee522086.png)

<Project Sdk="Microsoft.NET.Sdk.Razor">
  <PropertyGroup>
    <OutputType>WinExe</OutputType>
    <TargetFramework>net7.0-windows</TargetFramework>
    <Nullable>enable</Nullable>
    <UseWindowsForms>true</UseWindowsForms>
    <ImplicitUsings>enable</ImplicitUsings>
  </PropertyGroup>
  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.Components.WebView.WindowsForms" Version="7.0.81" />
  </ItemGroup>
</Project>

![image](https://user-images.githubusercontent.com/46733900/233742948-12507a13-bc6f-47c1-b3d8-c25bbb81d334.png)

**Comment by novac42** [comment]
Could you please provide some more information about this?

- SDK and Runtime version 
- Operating System

**Comment by Terrence77** [comment]
I am running Windows 10 (21H2 OS build 19044.2846
Project Sdk="Microsoft.NET.Sdk.Razor">

**Comment by Terrence77** [comment]
@Shangminx Do you have any information about this issue?

**Comment by vickiez** [comment]
Hi @Terrence77, you can find the WebView2 Runtime version by checking the path of the WebView2 process in Task Manager (under Properties when you right click on the process).

I created a work item to track this internally, we'll let you know on this thread when there is an update.

**Comment by Terrence77** [comment]
@vickiez Is this information what you need?
"Microsoft.AspNetCore.Components.WebView.WindowsForms" Version="7.0.81"
WebView2 (1.0.1370.28)

If not, I would bet the issue is in any of the latest runtime versions?

Thank you for helping.

**Comment by vickiez** [comment]
@Terrence77 1.0.1370.28 is the SDK version, it helps us to also know the WebView2 Runtime version when we try to repro

**Comment by Terrence77** [comment]
Is this it?
![image](https://user-images.githubusercontent.com/46733900/235807194-0b1c4971-c780-4828-a7db-49888fd58a88.png)

**Comment by vickiez** [comment]
Yes, thank you for the info!

**Comment by Terrence77** [comment]
@vickiez When I run my app in a Blazor Server app, the Audio control works perfectly.  I looked at the runtime while running the Blazor app and it is the same as I posted above.

**Comment by vickiez** [comment]
Got it, thanks. FWIW, I wasn't able to repro with a regular WebView2 WinForms app with that SDK and Runtime. If the issue is in Maui you may want to also open an issue here: https://github.com/dotnet/maui/issues

**Comment by Terrence77** [comment]
1.  So are you saying when you are playing an audio stream, and you move the timeline forward it sticks to the timeline?  When I do it, the audio reverts back to the beginning.  

2. Can I upload a zip file of my simple project to demo the issue?

**Comment by vickiez** [comment]
Yes, setting audio.currentTime is working fine for me with our [WinForms sample app](https://github.com/MicrosoftEdge/WebView2Samples/tree/main/SampleApps/WebView2WindowsFormsBrowser). I am just adding an audio element and advancing the time. A repro project would be great, thanks! Feel free to modify our Winforms app as well.

**Comment by Terrence77** [comment]
@vickiez 
Attached is my sample code,  Everything is in the Counter.razor page and the Components/TpsAudioPlayer.razor files.
I deleted the sample mp3 file in the wwwroot/audio folder so adjust the path in the Counter.razor file.

[WinFormsAudio.zip](https://github.com/MicrosoftEdge/WebView2Feedback/files/11391681/WinFormsAudio.zip)

Thank you.

**Comment by Terrence77** [comment]
@vickiez 
The sample app you posted is not using the same control I am...I don't think.
This sample app is using:

<TargetFrameworks>net462</TargetFrameworks>
 <PackageReference Include="Microsoft.Web.WebView2" Version="1.0.1777-prerelease" />

I am using 
    <TargetFramework>net7.0-windows</TargetFramework>
<PackageReference Include="Microsoft.AspNetCore.Components.WebView.WindowsForms" Version="7.0.81" />

So the Audio element might have worked in that old project, but it does not work in the newer versions.

**Comment by Terrence77** [comment]
@vickiez, @Shangminx any update on running my code?

**Comment by vickiez** [comment]
Our Winforms app is actually using the newer version of the WebView2 SDK, 1.0.1777-prerelease. The BlazorWebView component in your app appears to be using the 1.0.1370.28 WebView2 SDK. 

I changed our Winforms app to use the same SDK but couldn't repro, so the issue may be in BlazorWebView. It's maintained in the repo I linked above

**Comment by Terrence77** [comment]
Ok, over in the Maui area?
Thanks.

**Comment by sarah-harissa** [comment]
I am facing same issue in samsung tizen tv apps, sometimes current time is not working, maybe because mp3 is long? did any one find the reason for this?

**Comment by Terrence77** [comment]
@sarah-harissa Check out this topic over in Maui Blazor
https://github.com/dotnet/maui/issues/14964
